docker/run-dev.sh

```diff
@@ -104,16 +104,17 @@ docker run --name "$CONTAINER_NAME" \
     -d \
     "${DOCKER_FLAGS[@]}" \
     -v "$VOLUME_NAME":/host \
+    --entrypoint "cchost" \
     "$DOCKER_TAG" --config /host/dev-config.json
 
 echo "Setting up python virtual environment."
 if [ ! -f "venv/bin/activate" ]; then
-    if command -v python3.10 &> /dev/null; then
-        PYTHON=python3.10
-    elif command -v python3.8 &> /dev/null; then
-        PYTHON=python3.8
+    if command -v python &> /dev/null; then
```

Comment:
I think we want to be specific as to which python version is being used, i.e. pyscitt is said to be compatible with python3.8+ so either we increase that number or we make sure we test with it to avoid some weird regressions.